tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -93,8 +94,9 @@ def test_5_snapshots(
         )
         check_guest_connections(microvm, path, vm_blob_path, blob_hash)
         # Test vsock host-initiated connections.
+        start_guest_echo_server(microvm)
         path = os.path.join(microvm.jailer.chroot_path(), VSOCK_UDS_PATH)
```

Comment:
Maybe start_guest_echo_server could return this path?

Reply:
good idea, done